recipes/pyvirtualdisplay-feedstock/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: pyvirtualdisplay
+  version: {{ version }}
+
+source:
+  fn: pyvirtualdisplay-{{ version }}.tar.gz
+  url:  https://github.com/ponty/PyVirtualDisplay/archive/{{ version}}.tar.gz
+  sha256: 2c6bf006de7b1e950fae264d337c4344540e9f96d76e46f763fbb53e18798317
+
+build:
+  skip: true  # [win]
```

Comment:
pyvirtualdisplay uses as back-ends: Xvfb, Xephyr, Xvnc 
those are not available on windows